GliaWidgets/Theme/Alert/MessageAlertConfiguration.swift

```diff
@@ -1,11 +1,20 @@
 import SalemoveSDK
 
+/// Configuration of a generic alert.
```

Comment:
```suggestion
/// Configuration of the generic alert.
```

Reply:
Same as https://github.com/salemove/ios-sdk-widgets/pull/27#discussion_r654527648. I changed it to "the", but I feel like that's not correct here.

Reply:
Actually, you might be right here - I didn't fully understand the set of the alerts. Sorry about the noise!